tools/rtrclient.c

```diff
@@ -169,6 +367,10 @@ static void print_usage(char **argv)
 	printf("-k  Print information about SPKI updates.\n");
 	printf("-p  Print information about PFX updates.\n\n");
 
+	printf("-e export pfx table and exit\n");
```

Comment:
these are not reflected/shown in `usage` nor `examples` in the help message.

Also: please change port from `8282` to `8283` in examples to utilise the RIPE Validator v3

Reply:
I added examples for the export functionality and documented the `-o` option.